package/rmt-server.changes

```diff
@@ -1,3 +1,11 @@
+-------------------------------------------------------------------
+Wed May  9 09:36:02 UTC 2018 - wstephenson@suse.com
```

Comment:
Not sure if you want to update that date.

Reply:
not critical imo